include/libopencm3/nrf5/common/clock_common.h

```diff
@@ -0,0 +1,224 @@
+#define CLOCK_LFCLKSTAT                         MMIO32(CLOCK_BASE + 0x418)
+#define CLOCK_LFCLKSRCCOPY                      MMIO32(CLOCK_BASE + 0x41C)
+#define CLOCK_LFCLKSRC                          MMIO32(CLOCK_BASE + 0x518)
+#define CLOCK_CTIV                              MMIO32(CLOCK_BASE + 0x538)
+#define CLOCK_TRACECONFIG                       MMIO32(CLOCK_BASE + 0x55C)
+/**@}*/
+
+/* --- CLOCK_INTENSET values ----------------------------------------------- */
+/** @defgroup clock_intenset Enable interrupt
+@ingroup clock_defines
+@{*/
+#define CLOCK_INTENSET_HFCLKSTARTED                (1 << 0)
+#define CLOCK_INTENSET_LFCLKSTARTED                (1 << 1)
+#define CLOCK_INTENSET_DONE                        (1 << 3)
+#define CLOCK_INTENSET_CTTO                        (1 << 4)
+/**@}*/
+
+/* --- CLOCK_INTENCLR values ----------------------------------------------- */
+/** @defgroup clock_intenset Disable interrupt
+@ingroup clock_defines
+@{*/
+#define CLOCK_INTENCLR_HFCLKSTARTED                (1 << 0)
+#define CLOCK_INTENCLR_LFCLKSTARTED                (1 << 1)
+#define CLOCK_INTENCLR_DONE                        (1 << 3)
+#define CLOCK_INTENCLR_CTTO                        (1 << 4)
+/**@}*/
+
+/* --- CLOCK_HFCLKRUN values ----------------------------------------------- */
+/** @defgroup clock_hfclkrun Status indicating that HFCLKSTART task has been
+triggered
+@ingroup clock_defines
+@{*/
+#define CLOCK_HFCLKRUN_STATUS_TRIGGERED            (1 << 0)
+#define CLOCK_HFCLKRUN_STATUS_NOTTRIGGERED         (0 << 0)
+/**@}*/
+
+/* --- CLOCK_HFCLKSTAT values ---------------------------------------------- */
+/** @defgroup clock_hfclkstat HFCLK Status
+@ingroup clock_defines
+@{*/
+#define CLOCK_HFCLKSTAT_SRC_RC                     (0 << 0)
+#define CLOCK_HFCLKSTAT_SRC_XTAL                   (1 << 0)
+#define CLOCK_HFCLKSTAT_STATE_NOTRUNNING           (0 << 16)
+#define CLOCK_HFCLKSTAT_STATE_RUNNING              (1 << 16)
+/**@}*/
+
+/* --- CLOCK_LFCLKRUN values ----------------------------------------------- */
+/** @defgroup clock_lfclkrun Status indicating that LFCLKRUN task has been
+triggered
+@ingroup clock_defines
+@{*/
+#define CLOCK_LFCLKRUN_STATUS_TRIGGERED            (1)
+#define CLOCK_LFCLKRUN_STATUS_NOTTRIGGERED         (0)
+/**@}*/
+
+/* --- CLOCK_LFCLKSTAT values ---------------------------------------------- */
+/** @defgroup clock_lfclkstat LFCLK Status
+@ingroup clock_defines
+@{*/
+#define CLOCK_LFCLKSTAT_SRC_RC                     (0 << 0)
+#define CLOCK_LFCLKSTAT_SRC_XTAL                   (1 << 0)
+#define CLOCK_LFCLKSTAT_SRC_Xtal                   (1 << 0)
+#define CLOCK_LFCLKSTAT_SRC_SYNTH                  (0x2 << 0)
+#define CLOCK_LFCLKSTAT_SRC_Synth                  (0x2 << 0)
+#define CLOCK_LFCLKSTAT_STATE_NOTRUNNING           (0 << 16)
+#define CLOCK_LFCLKSTAT_STATE_NotRunning           (0 << 16)
+#define CLOCK_LFCLKSTAT_STATE_RUNNING              (1 << 16)
+#define CLOCK_LFCLKSTAT_STATE_Running              (1 << 16)
```

Comment:
Why do you need both of these?  I'd _really_ rather not have both, but if this has a good reason, please make the defines more like
```
#define WEIRD_NAME ORIGINAL_NAME
```
rather than duplicating the content

Reply:
The duplicate defines will be removed